app/Platform/RouteServiceProvider.php

```diff
@@ -63,8 +62,6 @@ protected function mapWebRoutes(): void
             // Route::get('create', CreateController::class)->name('create');
             // Route::resource('developers', DeveloperController::class)->only('index');
 
-            Route::get('docs/api', [ApiDocsController::class, 'index'])->name('docs.api');
```

Comment:
This page was meant for both web api and connect api docs in the future. But I'm not sure we'll ever need it 👍